build/pkgs/anyio/SPKG.rst

```diff
@@ -4,7 +4,11 @@ anyio: High level compatibility layer for multiple asynchronous event loop imple
 Description
 -----------
 
-High level compatibility layer for multiple asynchronous event loop implementations
+This spkg is installed from a wheel downloaded from
+PyPI.
+
+For a detailed description of the package, please visit its
+`PyPI page <https://pypi.org/project/anyio/>`_.
 
```

Comment:
These changes to SPKG.rst are not good. We build our manual from these! e.g. https://doc.sagemath.org/html/en/reference/spkg/appnope.html#spkg-appnope

Reply:
Of course the SPKGS could be reverted.  I almost did that.  But, after looking at many of them,  I decided that a link to the PyPI README file would be just as useful and in many cases more useful.  Most of those descriptions are written by copy-pasting from the PyPI README anyway.  These auto-generated SPKG.rst files provide the link.  A user reading the documentation who wanted to know the details about the package could simply click it.

So ultimately I decided that using the link was an improvement, and also that it clearly reduces the demands on the time of Sage developers.

Reply:
No, I disagree. It's intended that these pages in the documentation are readable rather than just being a collection of links. 
Actually I'd like to expand some of these descriptions with a description of *how* the package is being used in the Sage project. (That's part of a larger effort to improve the relation of Sage with upstream projects.)

Our `sage -package create` script does take the description from the PyPI README.
But some packages don't have a good description in their README, and we have some improvements. (Some of those should probably be upstreamed.)

By the way, we already have the PyPI link in the "Upstream Contact" section (if we don't, we should add it).

Reply:
Certainly a well-written description that explains how Sage uses the package would be much better than a link to PyPI.  It looks like a lot of work for one person to write those, but with a lot of people pitching in it might be feasible.  The link was the best I could do with the time I had available.  Feel free to revert or rewrite any of the descriptions!